ci/run_envoy_docker.sh

```diff
@@ -28,7 +28,7 @@ docker run --rm ${DOCKER_TTY_OPTION} -e HTTP_PROXY=${http_proxy} -e HTTPS_PROXY=
   -u "${USER}":"${USER_GROUP}" -v "${ENVOY_DOCKER_BUILD_DIR}":/build -v /var/run/docker.sock:/var/run/docker.sock ${GIT_VOLUME_OPTION} \
   -e BAZEL_BUILD_EXTRA_OPTIONS -e BAZEL_EXTRA_TEST_OPTIONS -e BAZEL_REMOTE_CACHE -e ENVOY_STDLIB -e BUILD_REASON \
   -e BAZEL_REMOTE_INSTANCE -e GCP_SERVICE_ACCOUNT_KEY -e NUM_CPUS -e ENVOY_RBE -e FUZZIT_API_KEY -e ENVOY_BUILD_IMAGE \
-  -e SYSTEM_PULLREQUEST_TARGETBRANCH \
+  -e ENVOY_SRCDIR -e ENVOY_BUILD_TARGET -e SYSTEM_PULLREQUEST_TARGETBRANCH \
```

Comment:
I don't think you need `ENVOY_SRCDIR`, because the source directory is mounted to `/source` anyway above, unless you change that too.

Reply:
The idea is that `/source` is where the script is run but that is different from where envoy code (`$ENVOY_SRCDIR`).

Reply:
Where are you mounting your source code to then? If you're using submodule this should be always `/source/envoy` otherwise you need to mount (which this script doesn't do)

Reply:
Yes, `/source/envoy`

Reply:
Running `envoy/ci/run_envoy_docker.sh 'ENVOY_SRCDIR=/source/envoy ENVOY_BUILD_TARGET=//:envoy envoy/ci/do_ci.sh bazel.dev'` works too but the change I'm proposing here makes CI setups easier to maintain (host env variables are separate from the actual command that is run).